tests/testenv/es_schema_creator.py

```diff
@@ -32,10 +32,22 @@
 DEPLOYMENT_SCHEMA = {
     'deployment': {
         'properties': {
-            'plan': {
```

Comment:
at the dark old times, we used to have the deployment plan under the deployment object; this was later removed - but apparently it was forgotten in this module :)